index.html

```diff
@@ -0,0 +1,85 @@
+<?php
+
+$db = new PDO('mysql:host=db'; 'dbname=cryptid_database','root','password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE);
+$query=$db->prepare("SELECT* FROM 'cryptids");
```

Comment:
Rather than doing select * you should select the specific fields you need. So SELECT `fieldname`, `otherfieldname` etc etc